doc/source/serve/model_composition.md

```diff
@@ -72,8 +79,53 @@ Composition lets you break apart your application and independently scale each p
 With composition, you can avoid application-level bottlenecks when serving models and business logic steps that use different types and amounts of resources.
 :::
 
+### ServeHandle Deep Dive
+
+Conceptually, ServeHandle is a client side load balancer, routing requests to any replicas of a given deployment. Also, it performs buffering internally so it won't overwhelm the replicas.
```

Comment:
ServeHandle is code-formatted (`ServeHandle`) throughout the rest of this documentation. I don't have a strong preference for code-formatted vs. not, but could we keep it consistent?

Reply:
formatting it